com.example.WebPConverter.json

```diff
@@ -0,0 +1,62 @@
+{
+  "app-id": "com.example.WebPConverter",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "45",
+  "sdk": "org.gnome.Sdk",
+  "command": "webp_converter.py",
+  "finish-args": [
+    "--share=network",
+    "--share=ipc",
+    "--socket=x11",
+    "--socket=wayland",
+    "--device=dri",
+    "--filesystem=home",
+    "--filesystem=xdg-config/gtk-3.0:ro",
+    "--filesystem=xdg-config/gtk-4.0:ro",
```

Comment:
does it use Gtk 3 or Gtk 4 Either way neither of this filesystem permmission is appropriate.

Reply:
removed.